.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: java
+install:
+- mvn install -P travis -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
The `-P travis` ensures that the `travis` profile is invoked from the `pom.xml` file [1][2], which I configured not to build the `cpp-build` module.
This command will run by default during the "install" phase by Travis, without the `-P travis` though by default [3][4]. I explicitly added it in this file to add the argument.

Do you think I should add a comment in this file? I think I can do that with a `#`.

[1] http://stackoverflow.com/questions/8304110/skip-a-submodule-during-a-maven-build
[2] http://maven.apache.org/guides/introduction/introduction-to-profiles.html
[3] https://docs.travis-ci.com/user/languages/java#Dependency-Management
[4] https://docs.travis-ci.com/user/customizing-the-build#The-Build-Lifecycle